.github/workflows/continuous-integration.yml

```diff
@@ -0,0 +1,16 @@
+name: CI
+
+"on":
+  push:
+    branches: [main]
+  pull_request:
+
+jobs:
+  test:
+    name: Test
```

Comment:
Could maybe rename this to something clearer so we can see at a glance in the GH checks `Linting`?

`Test` looks a little vague having just spotted this:

![image](https://user-images.githubusercontent.com/3327997/125761893-2955d810-0e16-45a3-b951-72afd6f39e80.png)

Reply:
Good point, I copied that from somewhere else and forgot to rename it!